<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schemaDialog</class>
 <widget class="QDialog" name="schemaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schema Editor</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1_2">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="8">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>0-7</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clr0Button">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="clr1Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="clr2Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="clr3Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="clr4Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="clr5Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QPushButton" name="clr6Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="8">
      <widget class="QPushButton" name="clr7Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="textLabel2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>8-15</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="clr8Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="clr9Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="clr10Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="clr11Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="clr12Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="clr13Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QPushButton" name="clr14Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QPushButton" name="clr15Button">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="iconSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="8">
      <widget class="QListWidget" name="nameListWidget"/>
     </item>
     <item row="4" column="1" colspan="8">
      <layout class="QHBoxLayout">
       <item>
        <widget class="QPushButton" name="saveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>Remove </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Background</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="noneRadioButton">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLineEdit" name="imageLineEdit"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="chooseButton">
          <property name="text">
           <string>Choose</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QRadioButton" name="imageRadioButton">
          <property name="text">
           <string>Image</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="optionComboBox">
          <item>
           <property name="text">
            <string>Tile</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Center</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Scale</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="coverComboBox">
          <item>
           <property name="text">
            <string>Whole</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Padding</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="4" column="1">
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="TextLabel2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="alphaSlider">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextLabel3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="alphaCheckBox">
          <property name="text">
           <string>Alpha</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Render</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Coverage</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="ForAlignment">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="ForAlignment_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Note:
 Alpha will only apply on background
 If Coverage is set to Padding
 Alpha will take no effect
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
